pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -696,11 +712,18 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podStatus *ku
 			}
 		}
 		if len(podContainerChanges.ContainersToUpdate[rName]) > 0 {
-			if err = m.updatePodContainerResources(pod, rName, podContainerChanges.ContainersToUpdate[rName]); err != nil {
+			if err = m.updatePodContainerResources(pod, rName, podContainerChanges.ContainersToUpdate[rName], false); err != nil {
 				klog.ErrorS(err, "updatePodContainerResources failed", "pod", format.Pod(pod), "resource", rName)
 				return err
 			}
 		}
+		if len(podContainerChanges.InitContainersToUpdate[rName]) > 0 {
+			if err = m.updatePodContainerResources(pod, rName, podContainerChanges.InitContainersToUpdate[rName], true); err != nil {
+				klog.ErrorS(err, "updatePodContainerResources failed for init containers", "pod", format.Pod(pod), "resource", rName)
```

Comment:
```suggestion
				klog.ErrorS(err, "updatePodContainerResources failed for init container", "pod", format.Pod(pod), "resource", rName)
```

Also I wonder why we do not include the container name in the error message

Reply:
We have logged the container name at line 816 of `updatePodContainerResources` since this function is updating resources for all containers and not just a single container. Updated the function name from `updatePodContainerResources` ---> `updatePodContainersResources` to make it clear.